frappe/utils/typing_validations.py

```diff
@@ -117,6 +118,9 @@ def transform_parameter_types(func: Callable, args: tuple, kwargs: dict):
 			continue
 		elif any(isinstance(x, ForwardRef | str) for x in getattr(current_arg_type, "__args__", [])):
 			continue
+		# ignore unittest.mock objects
+		elif isinstance(current_arg_value, mock.AsyncMock | mock.MagicMock | mock.Mock):
```

Comment:
```suggestion
		elif isinstance(current_arg_value, mock.Mock):
```

This should still work, other two are subclasses.

Reply:
@ankush good catch, I've added a commit that just uses the Mock class.